CRM/Core/Smarty.php

```diff
@@ -138,6 +138,11 @@ private function initialize() {
     }
 
     $this->register_function('crmURL', ['CRM_Utils_System', 'crmURL']);
+    $this->register_modifier('escape', ['CRM_Core_Smarty', 'escape']);
+
+    if (CRM_Utils_Constant::value('CIVICRM_SMARTY_DEFAULT_ESCAPE')) {
+      $this->default_modifiers = ['escape:"htmlall"'];
```

Comment:
Should this be appending to the default_modifiers array, in case it's already set?

Reply:
yeah makes sense